### PR TITLE
Add possibility to upload image to users and services

### PR DESCRIPTION
**Pull request by ritamma**

### CODE REVIEW

File: app/models/service.rb [hunk 1]
```diff
@@ -2,12 +2,19 @@ class Service < ApplicationRecord
...
   validates :description, presence: true, length: { maximum: 500 }
   validates :duration, presence: true, numericality: { greater_than: 0, only_integer: true }
   validates :price, presence: true, numericality: { greater_than: 0 }
 
+  before_save :set_image_url
+
+  private
+
+  def set_image_url
+    self.image_url = image&.url&.split('?')&.first
```
Comment (fooooxmr) [inline app/models/service.rb]: Use ActiveStorage method insted.

File: app/models/user.rb [hunk 2]
```diff
@@ -32,4 +34,11 @@ def active_for_authentication?
   def inactive_message
     !deleted_at ? super : :deleted_account
   end
+
+  private
+
+  def set_avatar_url
+    avatar_url = avatar&.url&.split('?')&.first
```
Comment (fooooxmr) [inline app/models/user.rb]: Use ActiveStorage method insted.

File: db/migrate/20230504140944_add_image_url_for_services.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class AddImageUrlForServices < ActiveRecord::Migration[7.0]
+  def change
+    add_column :services, :image_url, :string
```
Comment (fooooxmr) [inline db/migrate/20230504140944_add_image_url_for_services.rb]: Not needed.

**Review by fooooxmr — CHANGES_REQUESTED** [review]

File: app/models/service.rb (hunk 1 above)
Comment (ritamma) [reply app/models/service.rb]: Done

File: app/models/user.rb (hunk 2 above)
Comment (ritamma) [reply app/models/user.rb]: Done

File: db/migrate/20230504140944_add_image_url_for_services.rb (hunk 3 above)
Comment (ritamma) [reply db/migrate/20230504140944_add_image_url_for_services.rb]: Done

**Review by fooooxmr — APPROVED** [review]

**Pull request merged**